next-tavla/src/Shared/components/TransportIcon/index.tsx

```diff
@@ -21,7 +23,15 @@ function TransportIcon({
     )
 }
 
-export function getTransportIcon(transportMode: TTransportMode) {
+export function getTransportIcon(
+    transportMode: TTransportMode,
+    transportSubmode?: TTransportSubmode,
+) {
+    switch (transportSubmode) {
+        case 'localCarFerry' || 'nationalCarFerry' || 'regionalCarFerry':
+            return CarferryIcon
+    }
```

Comment:
Skjønner hvorfor dette er en switch for fremtiden, men syntes vi like gjerne kan ha det som en `if` for nå